script/update_sanctions_csv

```diff
@@ -35,15 +36,14 @@ while (my $row = $csv->getline($fh)) {
 
 # write
 my %names = map { $_ => 1 } @names;
-@names = sort(keys %names); # usort
+@names = sort(keys %names);    # usort
 
-my $csv_filepath = $INC{'Data/Validate/Sanctions.pm'};
-$csv_filepath =~ s/\.pm$/\.csv/;
-chmod 0666, $csv_filepath;
-open(my $out_fh, '>', $csv_filepath) or die "Can't write to $csv_filepath: $!\n";
+my $new_sanction_file = $sanction_file . ".tmp";
+open(my $out_fh, '>', $new_sanction_file) or die "Can't write to $new_sanction_file: $!\n";
```

Comment:
Although our current sources are ASCII only, might be worth adding UTF-8 support, it'd make it easier to use custom lists for other countries.